Makefile.am

```diff
@@ -224,6 +224,23 @@ clean-local::
 maintainer-clean-local::
 	rm -rf dist/ node_modules/ package-lock.json
 
+BUILT_SOURCES += package-lock.json
```

Comment:
Does that obsolete adding it to `EXTRA_DIST`? (It doesn't hurt, just curious)

Reply:
No.  These are two different things (as discussed).